### PR TITLE
FIX: disambiguation of staking client command

### PR DESCRIPTION
**Pull request by RevelationOfTuring**
Close: #185 

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/gov/keeper/tally_test.go [hunk 1]
```diff
@@ -242,10 +242,10 @@ func TestTallyDelegatorInherit(t *testing.T) {
 
 	coin, err := sdk.ParseDecCoin("11000" + common.NativeToken)
 	require.Nil(t, err)
-	delegator1Msg := staking.NewMsgDelegate(Addrs[3], coin)
+	delegator1Msg := staking.NewMsgDeposit(Addrs[3], coin)
 	stakingHandler(ctx, delegator1Msg)
 
-	voteMsg := staking.NewMsgVote(Addrs[3], []sdk.ValAddress{sdk.ValAddress(Addrs[2])})
+	voteMsg := staking.NewMsgAddShares(Addrs[3], []sdk.ValAddress{sdk.ValAddress(Addrs[2])})
```
Comment (lingtingfu) [inline x/gov/keeper/tally_test.go:248]: voteMsg := staking.NewMsgAddShares

the returned variable should be changed to 'addSharesMsg'

Comment (RevelationOfTuring) [reply x/gov/keeper/tally_test.go:248]: appreciated!

File: x/staking/keeper/proxy.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/okex/okchain/x/staking/types"
 )
 
-// ClearProxy clears the ProxyAddress on binded delegators
+// ClearProxy clears the ProxyAddress on the delegators who has bound
```
Comment (xiangjianmeng) [inline x/staking/keeper/proxy.go]: should change to " // ClearProxy clears the ProxyAddress on the delegator who has bound"

Comment (RevelationOfTuring) [reply x/staking/keeper/proxy.go]: Thx!

**Pull request merged**